python/lsst/ts/ATWhiteLightSource/wlsCSC.py

```diff
@@ -643,15 +659,22 @@ async def telemetryLoop(self):
 
             await asyncio.sleep(self.telemetry_publish_interval)
 
-    async def close(self):
-        self.interlockLoopBool = False
-        self.telemetryLoopBool = False
-        self.kiloarcListenerLoopBool = False
-        self.kiloarcModel.disconnect()
-        await self.chillerModel.disconnect()
-        await self.kilo_interloc
-        await self.telemetryLoopTask
-        await self.kiloarcListenerTask
+    async def close_tasks(self):
+        self.log.debug("Running csc close_tasks method")
+        await self._common_close()
         self.kiloarcModel.cooldown_task.cancel()
         self.kiloarcModel.warmup_task.cancel()
-        await super().close()
+        await super().close_tasks()
+
+    async def _common_close(self):
+        """
+        closes the high level tasks and disconnects from models, but doesn't
```

Comment:
Thanks for the doc string. But...please make it active voice and start it on the opening """ line. Both of those rules apply to all doc strings. For example here:
```
"""Close the high level tasks and disconnect from models, but do not
kill the safety timers.
"""
```